types/responselike/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for responselike 1.0
+// Project: https://github.com/lukechilds/responselike#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
```

Comment:
I don't see any node specific types here - is this needed?

Reply:
`'http'` and `'stream'` two lines below?